Assets/Scripts/UI/HealthBar.cs

```diff
@@ -0,0 +1,19 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthBar : MonoBehaviour
+{
+
+    public Slider slider;
```

Comment:
- [ ] Change `public` to `[SerializeField] private`. (Also, line 6 is an extra newline.)